openmc/mesh.py

```diff
@@ -210,3 +227,90 @@ def get_mesh_xml(self):
             subelement.text = ' '.join(map(str, self._width))
 
         return element
+
+    def build_cells(self, bc=['reflective'] * 6):
+        """Generates a list of cells which mimic the mesh geometry.
+
+        Parameters
+        ----------
+        bc : iterable of {'reflective', 'periodic', or 'vacuum'}
+            Boundary conditions for each of the six faces of a parallelopiped:
+            North, East, South, West, Up, and Down.
+
+        Returns
+        -------
+        root_cell : openmc.Cell
+            The cell containing the lattice mimicking the mesh geometry.
+        cells : iterable of openmc.Cell
+            The list of cells within each lattice position mimicking the mesh
+            geometry.
+
+        """
+
+        cv.check_length('bc', bc, length_min=6, length_max=6)
+        for entry in bc:
+            cv.check_value('bc', entry, ['transmission', 'vacuum',
+                                         'reflective', 'periodic'])
+
+        if len(self.dimension) == 2:
+            twod = True
+        else:
+            twod = False
+
+        # Build enclosing cell
```

Comment:
Done in the code, but for sake of this comment: its the cell which will hold/contain the lattice universe itself.